packages/next/src/client/components/react-dev-overlay/ui/components/errors/dev-tools-indicator/next-logo.tsx

```diff
@@ -466,7 +434,12 @@ export const NextLogo = forwardRef(function NextLogo(
                 <div>
                   Issue
                   {issueCount > 1 && (
-                    <span aria-hidden data-issues-count-plural>
+                    <span
+                      aria-hidden
+                      data-issues-count-plural
+                      // This only needs to animate when count changes from 1 -> 2
+                      data-animate={newErrorDetected && issueCount === 2}
```

Comment:
is it sth that we need when number increases? only checking for ===2 seems interesting  👀

Reply:
It animates the state change from 1 Issue → 2 Issue**s**